From f09395ee4b7159d37507c84a935cc67310726ab4 Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Thu, 24 Sep 2015 11:16:18 +0800
Subject: [PATCH] MLK-11613 ARM: dts: imx6qp-sabresd: Enable prefetch engines

This patch enables the four prefetch engines on the imx6qp-sabresd platform.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
---
 arch/arm/boot/dts/imx6qp-sabresd.dts | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qp-sabresd.dts b/arch/arm/boot/dts/imx6qp-sabresd.dts
index 87bee6a..7040ecd 100644
--- a/arch/arm/boot/dts/imx6qp-sabresd.dts
+++ b/arch/arm/boot/dts/imx6qp-sabresd.dts
@@ -12,3 +12,19 @@
 / {
 	model = "Freescale i.MX6 Quad Plus SABRE Smart Device Board";
 };
+
+&pre1 {
+	status = "okay";
+};
+
+&pre2 {
+	status = "okay";
+};
+
+&pre3 {
+	status = "okay";
+};
+
+&pre4 {
+	status = "okay";
+};
-- 
1.8.0

